src/schedule/dma_single_chan_domain.c

```diff
@@ -190,6 +199,9 @@ static int dma_single_chan_domain_register(struct ll_schedule_domain *domain,
 		if (data->channel->period == channel->period)
 			return 0;
 
+		trace_ll("dma_single_chan_domain_register(): lower period "
```

Comment:
I'd suggest you split into lines after the ":", like `trace_ll("dma_single_chan_domain_register(): " "lower period detected, registering again");`.